pkg/osquery/table/launcher_info.go

```diff
@@ -66,6 +75,22 @@ func generateLauncherInfoTable(db *bbolt.DB) table.GenerateFunc {
 			},
 		}
 
+		// No logger, so just ignore errors. generate the pem encoding if we can.
+		if eccKey := agent.Keys.Public(); eccKey != nil {
+			var pem bytes.Buffer
+			if err := osquery.PublicKeyToPem(eccKey, &pem); err == nil {
+				results[0]["signing_key"] = pem.String()
+				results[0]["signing_key_source"] = "TBD"
+			}
+		}
+
+		if localKey := agent.LocalDbKeys.Public(); localKey != nil {
```

Comment:
same here, prefer to key package level var unexported and get data via a func.

Reply:
(resolving this, because we only need one place to talk about it)